test/InterversionTests/SmugglerTests.cs

```diff
@@ -529,5 +530,127 @@ private static string GetSorter(string name)
             using (var reader = new StreamReader(stream))
                 return reader.ReadToEnd();
         }
+        
+        const string Server5Version = "5.0.0-nightly-20200401-0544";
```

Comment:
Maybe worth getting it dynamically to the latest nightly?

Reply:
Create a separate issue for this.